app/components/SpeakingTimeContent.tsx

```diff
@@ -47,10 +49,14 @@ export default function SpeakingTimeContent({
   return (
     <div>
       <ContentBlocks content={page.fields.content} locale={locale} />
+
       <details open={true} className="mx-auto max-w-7xl py-8 px-4">
```

Comment:
Would it make sense that when the user lands on the page, the filters section appears closed, so the user doesn't need to scroll so much to get to the coach cards (which is the main value of the page)? I would think it makes sense to display it closed first, then the 'filter anzeigen` label would make more sense, because on click we are 'displaying something', not hiding it as it happens now. It's also well-positioned and by using the icon, it will be even easier for users to identify it and use it if they need to filter through the coach list. 

WDYT @HerrBertling @Francine-Pepe ?
Markus, do you know if displaying it open was a specific request at some point made by the team?

Reply:
Yes, makes sense indeed, but the website displays it opened... if it´s ok to change, I am up for it.

Reply:
Sounds good 👍